cluster/examples/kubernetes/ceph/filesystem-ec.yaml

```diff
@@ -54,3 +54,4 @@ spec:
     #  requests:
     #    cpu: "500m"
     #    memory: "1024Mi"
+    priorityClassName:
```

Comment:
I find this confusing? What are the effects of leaving this blank? I think this would be better if commented out and given some sort of default value to serve as an example. Same for other instances of this.